docs/guides/authoring/Howlite/image/index.md

```diff
@@ -0,0 +1,40 @@
+# Image Component
+
+## Usage
+Drag & drop image component to a Page section. You will see a default image placeholder. (It is visible in edit mode only.)
+![Image placeholder](placeholder.jpg "Image placeholder in edit mode")
+
+## Authorable properties
+Click "Edit" icon to see all image component's properties:
+
+- You can open the left **assets panel** and drag&drop an image to the **L breakpoint** asset section. If only **L breakpoint** asset is provided, and __img__ html tag will be rendered with the provided image on all breakpoints
```

Comment:
Should we define what L, M, S breakpoint is? We clarify it the very first time a given letter is used

Reply:
@kamil-kwarciak I've added a link to breakpoints definition: https://github.com/websight-io/docs/pull/15/commits/86baba7f32807164e537a1e96991c837b3cfd2da